.gitignore

```diff
@@ -68,3 +68,6 @@ compiled/
 bin/
 include/
 src/
+
+# OSX
+.DS_Store
```

Comment:
Definitely doesn't matter much either way, but if you follow that point to the end, then every dev checks in ignore rules for their OS, editor, plugins, etc., for every project. Which IMO is kind of silly.